docs/firestore/usage.mdx

```diff
@@ -494,7 +494,24 @@ Future<void> addUser() {
 }
 ```
 
-Calling [`set`](!cloud_firestore.DocumentReference.set) with an id that already exists on the collection will replace all the document data.
+Calling [`set`](!cloud_firestore.DocumentReference.set) with an id that already exists on the collection will replace all the document data.  You can also specify `SetOptions(merge: true))` on the query, and this will merge the existing document with the data passed into the `set()`:
+
+```dart highlight={6-8}
```

Comment:
`dart highlight={6-8}` this would now be `dart highlight={7-11}` wouldn't it?

cc @Pingear @russellwheatley

Reply:
PR here: https://github.com/FirebaseExtended/flutterfire/pull/6630